api/src/db/Wish.js

```diff
@@ -1,19 +1,34 @@
 const mongoose = require('mongoose')
+const regex = /[{}+=_0-9\\|\[\])(*&^%%$#@!~`<>,?'";:]/;
 
 const Schema = mongoose.Schema
 
 const wishSchema = new Schema(
   {
     child: {
-      name: String,
+      name: { type: String,
+        validate: [
+          function(name){
+            return (!regex.test(name));
+          }, 'Name must contain only letters of the alphabet'
+        ],
+        required: [true, 'Child\'s name required']
+        },
       hometown: String,
       illness: String,
       age: Number
     },
     type: { type: String, enum: ['go', 'meet', 'be', 'have'] },
     details: String,
     sponsor: {
-      name: String,
+      name: { type: String,
+        validate: [
```

Comment:
I think that sponsors can definitely have number's and/or symbols in their name. For instance, Make-a-Wish itself has hyphens in it's name. Thoughts?

Reply:
Hyphens are in the allowed special characters however numbers are currently not allow. I see what your saying though. If a company is a sponsor they may have numbers in their name. I could add numbers to the sponsor allowed special characters or just pull the regex validation completely for symbols and number in the case of "sponsors".